showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITUnaryCallable.java

```diff
@@ -100,11 +101,38 @@ public void testGrpc_shutdown() {
   }
 
   @Test
-  public void testHttpJson() {
+  public void testHttpJson_receiveContent() {
     assertThat(echoHttpJson("http-echo?")).isEqualTo("http-echo?");
     assertThat(echoHttpJson("http-echo!")).isEqualTo("http-echo!");
   }
 
+  /*
+  This tests has the server return an error back as the result.
+  We use 404 NOT_FOUND Status as that has the same gRPC <-> HttpJson code mapping
+  (showcase sever has a map that translates the code). The showcase server expects
+  a gRPC Status Code and the result is the HttpJson's mapped value
+   */
+  @Test
+  public void testHttpJson_serverResponseError_throwsException() {
+    StatusCode.Code notFoundStatusCode = StatusCode.Code.NOT_FOUND;
```

Comment:
I just picked NOT_FOUND instead of CANCELLED since the docs seem to suggest that CANCELLED status is from a user action (https://chromium.googlesource.com/external/github.com/grpc/grpc/+/refs/tags/v1.21.4-pre1/doc/statuscodes.md). Any error code *should* be fine since we just need the server to respond back with the error.

Reply:
Whether we choose `NOT_FOUND` or `CANCELLED`, let's please make this and `testGrpc_serverResponseError_throwsException` consistent so we aren't implying a difference in behavior. If we want to test both behaviors, let's do so explicitly.

Reply:
I'll update to CANCELLED.